packages/validation/src/nodebalancers.schema.ts

```diff
@@ -38,7 +44,15 @@ export const nodeBalancerConfigNodeSchema = object({
   address: string()
     .typeError('IP address is required.')
     .required('IP address is required.')
-    .matches(PRIVATE_IPv4_REGEX, 'Must be a valid private IPv4 address.'),
+    .matches(PRIVATE_IPV4_REGEX, PRIVATE_IPV4_WARNING),
+
+  subnet_id: number().when('vpcs', {
+    is: (vpcs: typeof createNodeBalancerVPCsSchema) => vpcs !== undefined,
```

Comment:
Do we need this to be:

```suggestion
    is: (vpcs: typeof createNodeBalancerVPCsSchema[]) => vpcs !== undefined,
```

since the `vpcs` field is an array?

Reply:
Oops! Missed this. Thanks!